### ISSUE TITLE
test_legend_dots fails on matplotlib 1.4.3

### ISSUE THREAD

**Issue by QuLogic** [mpld3/mplexporter#38]
After applying #37, there is one more failure in the above test. It looks like the result is a little different:
```
opening figure
  opening axes
    draw line with 3 points
    draw 3 markers
    opening legend
    draw text 'None' None
    draw line with 2 points
    draw text 'label' None
    draw 2 markers
    draw text 'dots' None
    draw path with 4 vertices
    closing legend
  closing axes
closing figure
```

There's some extra text, and one less vertex in the last path.

**Comment by jakevdp** [comment]
Yes, currently we test with matplotlib 1.3. mpld3 relies on the private class hierarchy, which changed from 1.3 to 1.4 (and will likely change again with 1.5). The current package maintainers have not had the bandwidth to catch all these changes and corner cases.

There doesn't seem to be motivation/time among mpld3 developers and users to track-down and fix all these issues, so for the time being I'd suggest down-grading to matplotlib 1.3. Another option is for you to join the team and fix these issues yourself! In any case, once [MEP25](https://github.com/matplotlib/matplotlib/wiki/MEP25) happens, we can do a re-write of the package to use this utility and have a much more stable package.
